woocommerce/woocommerce
# Revert CYS modified version of edit-site lock-unlock module changes

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





It seems that GB reverted the content string change.  😂 

https://github.com/WordPress/gutenberg/blob/wp/6.4/packages/edit-site/src/lock-unlock.js#L8



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


1. Create a new JN site with this branch (using WooCommerce Beta Tester). Make sure to select `Include WordPress Beta Tester` under the `Plugins` section. 
2. Complete the core profiler and make sure to connect to Jetpack.
3. Go to `Tools -> WCA Test Helper -> Features` and enable `customize-store`
4. Go to `WooCommerce -> Home` and click on `Customize your store` task.
5. Go through the steps and make sure everything displays without an error.
6. Let's test with WP 6.4
7. Go to `Tools -> Beta Testing`
8. Choose `Bleeding edge` and `Beta/RC Only` then click on `Save Changes`
9. Go to `Dashboard -> Updates`
10. Choose 'Update to version 6.4` 
11. Start Customize your store task again and repeat the test.




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
